### DISCUSSION TITLE
Contributor call: 30 April 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2132] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @joostdecock and @kaermorhan this time, since both @tangerineshark and @lexander2002 will be absent.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 30 April 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/apr-30-2022/4pm).
Or check the bot in Discord! #meetings
The meetings will always be at 6pm, Antwerp time.

### Agenda
- #2024 
- **@eleonoremilbou reached out to a wheelchair-bound sewist and we should come up with some questions to ask them**
- [**Lerna is deprecated**](https://github.com/lerna/lerna/#readme)
- **State of the Union** 

> This discussion will be updated after the call with notes and to-dos.

**Comment by TriploidTree** [comment]
I'd like to add https://github.com/freesewing/freesewing/issues/2024#issue-1178646056 if there's time.

*adds calendar reminder*

**Reply by joostdecock** [reply]
Done :)

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:
- @joostdecock
- @bobgeorgethe3rd 
- @TriploidTree 
- @woutervdub 
- Jackseye
- @eleonoremilbou
- @KaerMorhan

**TriploidTree**
Further pattern customization:
- How to customize it for yourself
- Imagining the end result of the pattern

Knowing calculations and logic behind them (designer’s notes) to better customize the pattern.
Suggestion: pattern notes/reviews for specific body types, maybe in a clickable link format; possibly later implementing into the website programming.
_Bob’s feedback: cooking blogs, Victorian sewing manuals which provide additional information for pattern adjusting for different bodies._

**Eleonore**
Optimizing patterns for wheelchair users.
Jacqueline (Eleonore’s acquaintance) – a Dutch wheelchair dancer who saws her own costumes – agreed to give her feedback (maybe join the next Contributor Call).

**Additional suggestion:** include disabled people in illustrations showing patterns’ final results.

Questions to ask:
- Is there something Jacqueline has always wanted to make/wear but couldn’t because the clothes for wheelchair users aren’t fitted for that?
Example: long coats.

Eleonore has a list of questions. Joost will also ask his college to give their feedback and also maybe join the next Contributor Call.
_Bob’s feedback: magnetic snaps for people with lower mobility._

**Joost**
Large amount of stress and work leave little time to make high-quality projects that give the feeling of fulfillment.
Suggestions:
- taking on less tasks and polishing them to a better level;
- more clear instructions and plans for features to implement;
- splitting the work among more people and delegating some of the tasks to other members (more stable features);
- going back to the basics to improve UX;
- “change a typo” type of work for users.

_Eleonore’s feedback: applying the rule of building a platform that brings other people joy for yourself.
Bob’s feedback: action plans! Going through the main list of all the tasks so people could volunteer to work on specific tasks. Taking over markdown, UI and UX.
Jackseye’s feedback: creating a tutorial on commits, suggesting ideas/implementing new features for new people who want to help but don’t know where to start._

**Bobgeorgethe3rd**
Updates and improvements on Billy Bucket Hat.

**Reply by l3xander** [reply]
WOWWWWW I love how you did these notes

**Reply by KaerMorhan** [reply]
Thank you!
Tried my best :)

**Reply by SeaZeeZee** [reply]
Seriously well done I got a good idea of what was discussed and I wasn't even there!

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
